api/src/services/opportunities_v1/search_opportunities.py

```diff
@@ -127,7 +127,13 @@ def _add_search_filters(
             builder.filter_int_range(field_name, field_filters.min, field_filters.max)
 
         elif isinstance(field_filters, DateSearchFilter):
-            builder.filter_date_range(field_name, field_filters.start_date, field_filters.end_date)
+            builder.filter_date_range(
+                field_name,
+                field_filters.start_date,
+                field_filters.end_date,
+                field_filters.start_date_relative,
+                field_filters.end_date_relative,
```

Comment:
If you go with the idea I had on the query builder, this would become something like:
```
start_date=field_filters.start_date if field_filters.start_date else field_filders.start_date_relative,
end_date=field_filters.end_date if field_filters.end_date else field_filders.end_date_relative,
```